ephios/core/models/users.py

```diff
@@ -527,6 +527,9 @@ def body(self):
     def is_obsolete(self):
         return self.notification_type.is_obsolete(self)
 
+    def __str__(self):
+        return f"{self.subject} for {self.user or 'Guest'}"
```

Comment:
translate

Reply:
and remember we can't translate f-strings :sweat_smile:

Reply:
yeah didn't want to translate it because it never shows in any UI, thats why f-string ^^